.gitignore

```diff
@@ -164,3 +164,7 @@ $RECYCLE.BIN/
 # Aditional custom ignores
 
 .settings/
+
+node_modules
```

Comment:
A comment before these explain _why_ we're ignoring these might be good. Otherwise someone might (in a quick skim) think we're doing The Bad Thing and ignoring things like `package.json` that we "shouldn't" ignore.